README.md

```diff
@@ -16,7 +16,6 @@ near <command>
 ### Commands:
 ```Bash
   near create_account  # create a developer account
-  near build           # build your smart contract
   near deploy          # deploy your smart contract
```

Comment:
Why remove? I personally like this syntax more

Reply:
do you mean like just make it alias for `gulp` or like keep it using own gulpfile which user cannot modify without forking `near-shell` (i.e. avoid doing https://github.com/nearprotocol/near-shell/issues/4)?

Reply:
Brought back `near build` as `gulp` alias